### PR TITLE
SERVICE: Attributes folder and inside it UserPassword class created. #65

### LINKED ISSUE #65: SERVICE: create UserPassword class inside Attributes foldier
This class check for strong password with including numbers letters and characters.

### PR DESCRIPTION
**Pull request by OlimSadullaev**
Closes #65

### CODE REVIEW

**Comment by OlimSadullaev** [comment]
UserPassword created

**Comment by OlimSadullaev** [comment]
Created

File: BelissimoCloneWPF.Service/Attributes/UserPassword.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+namespace BelissimoCloneWPF.Service.Attributes
+{
+    public class UserPassword : ValidationAttribute
+    {
+        public override bool IsValid(object value)
+            => value is string password &&
+                password.Length >= 8 &&
+                    password.Any(c => char.IsDigit(c)) &&
+                        password.Any(c => char.IsLetter(c));
```
Comment (MuhammadKasimov) [inline BelissimoCloneWPF.Service/Attributes/UserPassword.cs:16]: override IsValid which returns ValidationResult

**Review by MuhammadKasimov — CHANGES_REQUESTED** [review]

**Comment by ShohruhUzDev** [comment]
Tag issue

File: BelissimoCloneWPF.Service/Attributes/UserPassword.cs [hunk 2]
```diff
@@ -9,10 +10,32 @@ namespace BelissimoCloneWPF.Service.Attributes
...
-                password.Length >= 8 &&
+        protected override ValidationResult? IsValid(object value, ValidationContext validationContext)
+        {
+            var values = (string)value;
+            
+            if (!(values is string password && password.Length >= 8 &&
                     password.Any(c => char.IsDigit(c)) &&
-                        password.Any(c => char.IsLetter(c));
+                        password.Any(c => char.IsLetter(c))))
+                return new ValidationResult("password must be at least one number and at least one letter and number of characters more or equal to 8");
```
Comment (MuhammadKasimov) [inline BelissimoCloneWPF.Service/Attributes/UserPassword.cs]: Password not password

**Review by MuhammadKasimov — CHANGES_REQUESTED** [review]

File: BelissimoCloneWPF.Service/Attributes/UserPassword.cs (hunk 2 above)
Comment (OlimSadullaev) [reply BelissimoCloneWPF.Service/Attributes/UserPassword.cs]: Changes Accomplished from password to Password

**Pull request merged**